test/fixtures/merge_sub_file.yaml

```
@@ -0,0 +1,109 @@
+subs:
+-
+  hostname: .*
+  type: Hypervisor
+  match:
```

after https://github.com/RedHatSatellite/katello-attach-subscription/pull/41 this needs to be:

```suggestion
  facts:
```